### ISSUE TITLE
Available Users

### ISSUE THREAD

**Issue by runninfrom** [iw4p/Ton-Fragment#2]
![image](https://user-images.githubusercontent.com/131925044/234723613-79e4fa6a-8874-4f49-bf6d-e1a5ac102f23.png)

Hey,

I was wondering if there would be a way to find available users that a person can bid on. If there is a way to do this, I would appreciate a code reference as I have been unable to figure out a way. 

Thanks.

**Comment by iw4p** [comment]
Hello. 
In Usernames section in Readme file, I provided some ways and snippets to this job. Have you tried them?

**Comment by runninfrom** [comment]
Hi again,
Yes, I have read everything but I'll read it again. Apologies beforehand if I didn't see it and it was there.
Thanks.

**Comment by runninfrom** [comment]
Yes, I can confirm that nothing there has worked. What I am looking for is to search for a word that is NOT up for auction but rather a word that is eligible for a person to leave a bid on it. I'm not sure if you get what I mean, but I'm searching for a way to search up a word and see if it is available for me to leave a bid on it.

**Comment by iw4p** [comment]
As I understood, you need a method to give it your specific username, and fetch its data to know is it on sale or not?

**Comment by runninfrom** [comment]
Apologies, I forgot to respond to this. 

Yes, that is exactly what I want.

**Comment by iw4p** [comment]
There is no feature that can do this for you currently, but you can explain exactly what data you need and which details are important to you and I'll add it for you in the next version. 
What's the essential elements for you to fetch when you give the ID the method?
